source/partials/_changelog.erb

```diff
@@ -4,6 +4,7 @@
 
 | Publication date | Update                                                                                                                                                                                                                                                                                                                                            |
 |------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Jan 13 2025      | Updates guidance <a href="/integrate-with-integration-environment/authenticate-your-user">"Authenticate your user"</a> to add information about configuring max_age parameter.                                                                 |
```

Comment:
suggest replace "configuring max_age parameter." with "configuring the `max_age` parameter